src/adapter/src/coord/sequencer/cluster.rs

```diff
@@ -540,21 +542,21 @@ impl Coordinator {
             .await;
     }
 
-    /// When this is called by the automated cluster scheduling, `scheduling_decision_reason` should
-    /// contain information on why is a cluster being turned On/Off. It will be forwarded to the
-    /// `details` field of the audit log event that records creating or dropping replicas.
     pub async fn sequence_alter_cluster_managed_to_managed(
         &mut self,
         session: Option<&Session>,
         cluster_id: ClusterId,
         config: &ClusterVariantManaged,
         new_config: ClusterVariantManaged,
         reason: ReplicaCreateDropReason,
-    ) -> Result<(), AdapterError> {
+        strategy: AlterClusterPlanStrategy,
+    ) -> Result<bool, AdapterError> {
         let cluster = self.catalog.get_cluster(cluster_id);
         let name = cluster.name().to_string();
         let owner_id = cluster.owner_id();
-        let mut ops = vec![];
```

Comment:
Why do the ops need to be split up? If there's a good reason, it should be documented in a comment here like "all drop ops must be run before all other ops" or something.

Reply:
I've added a comment, and attempted to clean this up a bit for clarity.